updater/scripts/git-download.sh

```diff
@@ -4,7 +4,7 @@
 #
 echo -e "repository\tuser\tcloning?\trequests/day\tdownload/day [B]"
 
-zcat -f /var/log/github/audit.log.1* |
+zcat -f /var/log/github-audit.log.1* |
```

Comment:
Can you make this variable based on the output here: https://github.com/larsxschneider/scotty/blob/a68d16339a6da8cd20559ce36533f148e3376d47/lib-ghe.sh#L66-L67

Reply:
Yes, Doing a version check is reasonable, will try to adapt it.